core/src/main/java/org/elasticsearch/index/termvectors/TermVectorsService.java

```diff
@@ -72,6 +72,7 @@
     private TermVectorsService() {}
 
     public static TermVectorsResponse getTermVectors(IndexShard indexShard, TermVectorsRequest request) {
+        final long startTime = System.currentTimeMillis();
```

Comment:
It's better to use a relative time source like `System#nanoTime` to calculate the length of time instead of an absolute time source (which is subject to crazy things like NTP adjustments, DST modifications, or just end-user changes).